client/src/components/AddPost/PostPost/index.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+
+export default class PostPhoto extends React.Component<any> {
```

Comment:
why we can't name this file -- postPhoto?

Reply:
answered in telegram